docker-compose.yml

```diff
@@ -23,7 +25,10 @@ services:
       timeout: 5s
       retries: 10
     environment:
+      - DB_HOST=host.docker.localhost
```

Comment:
Are you sure about this one?

```
host.docker.internal which resolves to the internal IP address used by the host. This is for development purpose and will not work in a production environment outside of Docker Desktop for Windows / Mac.
```

https://docs.docker.com/docker-for-mac/networking/

I would suggest to put this rather into docs/readme.md

Reply:
i did try ```host.docker.internal``` that made all the services UP and running, they were exiting with non-zero before, but i also found that the login stopped working, so i reverted back to not having ```host.docker.internal```